test/ssc_test/cmod_utilityrate5_test.cpp

```diff
@@ -1484,11 +1484,11 @@ TEST(cmod_utilityrate5_eqns, Test_Commercial_kWh_per_kW_charges_with_billing_dem
                                          11, 1, 9.9999999999999998e+37, 0 };
     ssc_data_set_matrix(data, "ur_dc_flat_mat", p_ur_dc_flat_mat, 12, 4);
 
-    ssc_data_set_number(data, "ur_ec_enable_billing_demand", 1);
-    ssc_data_set_number(data, "ur_ec_billing_demand_minimum", 100);
-    ssc_data_set_number(data, "ur_ec_billing_demand_lookback_period", 11);
+    ssc_data_set_number(data, "ur_enable_billing_demand", 1);
+    ssc_data_set_number(data, "ur_billing_demand_minimum", 100);
+    ssc_data_set_number(data, "ur_billing_demand_lookback_period", 11);
```

Comment:
Do you have a min, max for lookback months? (Should be taken care of in the compute module precheck.

Reply:
Yes, 0 and 12:

![image](https://user-images.githubusercontent.com/5530592/140963871-2b4d3895-e91d-4414-a50a-efac5a70b5a4.png)